collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2027,7 +2027,7 @@ public void onClick(DialogInterface dialog,
                                         languages[whichButton]);
                                 String selection = FormsColumns.FORM_FILE_PATH
                                         + "=?";
-                                String[] selectArgs = {formPath};
+                                String[] selectArgs = {StorageManager.getFormFilePath(StorageManager.getRelativeFormFilePath(formPath))};
```

Comment:
Because `getFormFilePath()` (final name `getFormFilePathToStoreInDatabaseBasingOnRelativePath()`) expects a relative path as a parameter. Theoretically I could have a method in which as a passed value I would accept both relative and absolute paths and then deal with it but I'm not sure it's better, somehow I decided to do that in this way. I'll look at it again.

Reply:
Fixed, now one method deals with it.